docs/source/intro/get_started.rst

```diff
@@ -0,0 +1,123 @@
+.. _intro/get_started:
+
+****************
+Getting Started
+****************
+
+Installation
+============
+
+There are numerous routes to setting up the full AiiDA environment, depending on your use case.
+Here we first provide the simplest approach for installation on your local computer.
+
+`Conda <https://docs.conda.io>`_  provides a cross-platform package management system, from which we can install the basic components of the AiiDA infrastructure in an isolated environment:
+
+.. code-block:: console
+
+    $ conda create -n aiida -c conda-forge aiida-core aiida-core.services
+    $ conda activate aiida
```

Comment:
This activates the services, correct? I think we should either move this into a separate code block or mention right after, that this will need to be run after every reboot. Ideally, we can avoid this all-together in the future, but for now we need to mention that.

Reply:
> This activates the services, correct?

No this is just activating the conda enviroment (aiida is the environment name, maybe that is misleading?). It does need to be run when "entering" a new terminal though, i.e. to set the environment

Reply:
Sorry, I completely misread this, sorry.